devel/apache-ant-1.9/Portfile

```diff
@@ -41,9 +43,10 @@ build.cmd               true
 variant source description "build from source; not recommended" {
         distname                        apache-ant-${version}-src
         master_sites.mirror_subdir      source
-        checksums                       md5    c988158e101e7700b45c14b9804fd554 \
-                                        sha1   4c815a6f560cde94fc2b3d15e34393ebf6dbca52 \
-                                        sha512 27fe030f134e34d4e06ef32baf1e8cf392490745fa7a7f47cf3e123f1141499c059e317154c923a8d7dcb7af163eb0cc87636f5d6150e0ce54b6c0abc2d0a112
+        checksums                       sha1    4c815a6f560cde94fc2b3d15e34393ebf6dbca52 \
```

Comment:
See other comment on sha1.

Reply:
I deliberately left it in to demonstrate that the file has not changed.